### DISCUSSION TITLE
Cybo-Drummer | 6/ 6 MIDI router/mapper for mapping drum triggers to drum computers | Humanize those drum computers!

### DISCUSSION THREAD

**Discussion by HLammers** [micropython/micropython#16192] [Show and tell]
I’m happy to announce the very first beta release of my project Cybo-Drummer!

All the code is written in MicroPython.

![](https://raw.githubusercontent.com/HLammers/cybo-drummer/refs/heads/main/images/cybo-drummer_v0.1.jpg)

# Introduction
I own an electronic drum kit and a bunch of drum computers and my dream was to use the former to play the latter, so I went searching for a way to do just that – allowing me to easily switch between different configurations combining the sounds of one or more drum computers. I looked for hardware solutions, but couldn’t find any. I looked for software solutions, but I could only find MIDI mappers or other complex solutions that would never give me the easy to use experience I had it mind. It turns out that (as usual) I go against the current fashion of trying to make an electronic drum kit sound (and look) as acoustic as possible. So I decided to develop my own solution – and publish it as open source DIY project, hoping it finds like-minded drummers!

# What it is
Cybo-Drummer is a MIDI router/mapper with 6 input ports and 6 output ports, specially designed for mapping drum triggers (electronic drum kits’ brains) to drum computers. Since there is no standard for the MIDI messages sent by drum kits, nor the messages received by drum computers, Cybo-Drummer offers a flexible way of mapping the one to the other.

# Link
More information on Cybo-Drummer, it’s schematics, Gerber files for the PCBs and source code are all available on GitHub: [https://github.com/HLammers/cybo-drummer](https://github.com/HLammers/cybo-drummer)

**Comment by bidrohini** [comment]
Thanks for sharing.

**Comment by keenanjohnson** [comment]
This is excellent!
